### PR TITLE
feat(openldap): Add OpenLDAP role

### PR DESCRIPTION
**Pull request by Ferenc-**

### CODE REVIEW

File: roles/openldap/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+docker_registry_openldap: quay.io
+
+##########################
+# openldap
+
+openldap_configuration_directory: /opt/openldap/configuration
+openldap_secrets_directory: /opt/openldap/secrets
+openldap_docker_compose_directory: /opt/openldap
+
+openldap_network: 172.31.100.32/28 # TODO: Find a proper one not shared with phpmyadmin
```
Comment (berendt) [inline roles/openldap/defaults/main.yml:20]: Check http://docs.osism.de/appendix/tables/service-networks.html. There is a table with all used network. The last assigned one is ``172.31.100.224/28``. The next free one is ``172.31.100.240/28``.

File: roles/openldap/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+openldap_container_name: openldap
+
+openldap_host: 192.168.16.5 # TODO: Double check this, perhaps use 127.0.0.1
```
Comment (berendt) [inline roles/openldap/defaults/main.yml:23]: Yes. Use ``127.0.0.1`` as default here.

Comment (Ferenc-) [reply roles/openldap/defaults/main.yml:23]: Done

File: roles/openldap/defaults/main.yml (hunk 1 above)
Comment (Ferenc-) [reply roles/openldap/defaults/main.yml]: Done

**Comment by berendt** [comment]
@Ferenc- Sorry for the delay. I hope to get to it by Monday.

File: roles/openldap/defaults/main.yml (hunk 2 above)
Comment (Ferenc-) [reply roles/openldap/defaults/main.yml]: So the way I understood, these settings here are overridden by the settings in 
[ansible-defaults](https://github.com/osism/ansible-defaults/blob/main/all/099-infrastructure.yml#L21).
But what I so far experienced is that this value somehow still remains on `127.0.0.1`, and doesn't take the `192.168.16.5` value that I would expect in pluscloud.
I tried to follow the keycloak way, but I can't find the difference, yet keycloak gets the right address  `192.168.16.5`.
Do you perhaps have a tip what might be still missing?

Comment (Ferenc-) [reply roles/openldap/defaults/main.yml]: Never mind, it must have been a transient problem. Now it's gone.

**Pull request merged**